litellm/model_prices_and_context_window_backup.json

```diff
@@ -1730,6 +1730,22 @@
         "supports_assistant_prefill": true,
         "supports_prompt_caching": true
     },
+    "claude-3-5-haiku-20241022": {
+        "max_tokens": 8192,
+        "max_input_tokens": 200000,
+        "max_output_tokens": 8192,
+        "input_cost_per_token": 0.000001,
+        "output_cost_per_token": 0.000005,
+        "cache_creation_input_token_cost": 0.00000125,
+        "cache_read_input_token_cost": 0.0000001,
```

Comment:
Not sure about than. Trusting Anthropic docs, this don't seems accurate.

![image](https://github.com/user-attachments/assets/2810a74c-efa0-4c6b-b904-e9b00d7be1ea)

Reply:
Reference to doc: https://docs.anthropic.com/en/docs/build-with-claude/prompt-caching#supported-models

Reply:
Ah makes senses then !

Reply:
I did write to Anthropic supports to address this.